app/views/shared/_navigation.html.erb

```diff
@@ -14,9 +14,13 @@
     <img src="https://voices-of-consent-static-images.s3.amazonaws.com/Design+Stage.svg" title="Design">
     <span>Design</span>
   </a>
+  <a class="nav-icon" href="<%= box_requests_path %>">
```

Comment:
Noting here that @abudri talked about this — it might be nice to change all the links here to use `<%= link_to %>` instead of this current pattern, and that's something that can wait for a future PRa